CHANGELOG.rst

```diff
@@ -152,6 +174,28 @@ New features
   a clearer way to indicate that PySys will automatically determine how many 
   threads to run tests with based on the number of available CPUs. 
 
+- Added `BaseTest.pythonDocTest()` method for executing the doctests in a 
+  Python file. 
+
+- Added `ProcessUser.compareVersions()` static helper method for 
+  comparing two alphanumeric dotted version strings. 
+
+- Added `ProcessUser.getDefaultEnvirons()` method which is now used by 
+  `startProcess()` to provide a minimal but clean set of environment variables 
+  for launching a given process, and can also be used as a basis for creating 
+  customized environments using the new `createEnvirons()` helper method. 
+  There are some new project properties to control how this works, which 
+  are added to the sample `pysysproject.xml` and recommended for new projects, 
+  but are not enabled by default in existing projects to maintain 
+  compatibility::
```

Comment:
Redundant double colon?

Reply:
Nope that's deliberate - this is a reStructured text file, and the double colon is how rst designates that the following block is a preformatted code snippet. As a result we get nice formatting when we view these files from github (and in future, if we end up generating html from them)

Reply:
And now I have learned two things today :-)